lib/matplotlib/tests/test_backends_interactive.py

```diff
@@ -128,6 +128,8 @@ def check_alt_backend(alt_backend):
         "matplotlib.backends.backend_{}".format(backend))
 
     ax.plot([0, 1], [2, 3])
+    if fig.canvas.toolbar:  # i.e toolbar2.
+        fig.canvas.toolbar.draw_rubberband(None, 1, 1, 2, 2)
```

Comment:
Shouldn't you try floats? Otherwise, this wouldn't have caught the problem here.

Reply:
Actually it would have worked because we use self.bbox.height - y0, but sure, testing some floats is good too.